vkcs/kubernetes/resource_cluster.go

```diff
@@ -142,9 +142,9 @@ func ResourceKubernetesCluster() *schema.Resource {
 					"  * `calico_ipv4pool` to set subnet where pods will be created. Default 10.100.0.0/16. _note_ Updating this value while the cluster is running is dangerous because it can lead to loss of connectivity of the cluster nodes.\n" +
 					"  * `clean_volumes` to remove pvc volumes when deleting a cluster. Default False. _note_ Changes to this value will be applied immediately.\n" +
 					"  * `cloud_monitoring` to enable cloud monitoring feature.\n" +
-					"  * `etcd_volume_size` to set etcd volume size. Default 10Gb.\n" +
+					"  * `etcd_volume_size` to set etcd volume size. Integer value, default 10 (Gb).\n" +
```

Comment:
It is string in fact, as we see in examples. So remove value type.
Probably "to set etcd volume size in GB. Default 10"?

Reply:
used your text added 2 defaults